tests/sentry_plugins/twilio/test_plugin.py

```diff
@@ -66,3 +71,36 @@ def test_is_configured(self):
         for o in ("account_sid", "auth_token", "sms_from", "sms_to"):
             self.plugin.set_option(o, "foo", self.project)
```

Comment:
@ceorourke it would also be good to check that `is_configured ` returns false until you have every option set. You could easily do that by moving the assertion on line 70 into the for loop before `set_option` is called.